build/cloudbuild_tools.yaml

```diff
@@ -12,22 +12,36 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+substitutions:
+  _KUBERNETES_VERSION: v1.19.2
 steps:
 - name: "gcr.io/cloud-builders/docker"
-  args: ["build", "-t", "gcr.io/kubebuilder/thirdparty-${_GOOS}:1.20.2", "./build/thirdparty/${_GOOS}"]
+  args: [
+    "build",
+    "--build-arg", "OS=${_GOOS}",
+    "--build-arg", "ARCH=${_GOARCH}",
+    "--build-arg", "KUBERNETES_VERSION=${_KUBERNETES_VERSION}",
+    "-t", "gcr.io/kubebuilder/thirdparty-${_GOOS}-${_GOARCH}:${_KUBERNETES_VERSION}",
+    "./build/thirdparty/${_GOOS}",
+  ]
   # darwin takes forever
   timeout: 30m
 
-- name: "gcr.io/kubebuilder/thirdparty-${_GOOS}:1.20.2"
-  args: ["cp", "/kubebuilder_${_GOOS}_${_GOARCH}.tar.gz", "/workspace/kubebuilder-1.20.2-${_GOOS}-${_GOARCH}.tar.gz"]
-  env:
-  - 'GOOS=${_GOOS}'
-  - 'GOARCH=${_GOARCH}'
-- name: 'gcr.io/cloud-builders/gsutil'
-  args: ['-h', 'Content-Type:application/gzip', 'cp', '/workspace/kubebuilder-1.20.2-${_GOOS}-${_GOARCH}.tar.gz', 'gs://kubebuilder-tools/kubebuilder-tools-1.20.2-${_GOOS}-${_GOARCH}.tar.gz']
-  env:
-  - 'GOOS=${_GOOS}'
-  - 'GOARCH=${_GOARCH}'
-images: ["gcr.io/kubebuilder/thirdparty-${_GOOS}:1.20.2"]
+- name: "gcr.io/kubebuilder/thirdparty-${_GOOS}-${_GOARCH}:${_KUBERNETES_VERSION}"
```

Comment:
should we keep something around that cross-tags this to the old `thirdparty-${OS}` format too, so we don't confuse people?  (this is a question, not a suggestion -- I can't quite convince myself either way)

Reply:
Is this image intended to be pulled/used outside of this build system? If so then I don’t think the old format should be kept around because it is now ambiguous.

Reply:
What about a docker manifest? I think it still requires experimental mode on docker in order to create them but nothing special is needed in order to get the images.